CHANGELOG.rst

```diff
@@ -12,9 +16,10 @@ User-Facing Changes
 
 **🐞 Bug Fixes**
 
+- **Search Services**: Resolve issues that caused items to be missing from Renku 2.0 search, including the in the search for adding members to projects and groups.
```

Comment:
```suggestion
- **Search Services**: Resolve issues that caused items to be missing from Renku 2.0 search, including in the search for adding members to projects and groups.
```

Reply:
I cannot process the second part of the sentence; do we want to give examples of items that were missing or we want to say that search now includes projects and groups' members. If the first case, I'd write it as `..., including  projects and groups' members` and if the second case: `... including adding members to projects and groups in the search`.